src/main/assembly/dist/cfg/application.properties

```diff
@@ -1,7 +1,7 @@
 owner={{ easy_stage_dataset_owner }}
-storage-base-url={{ easy_stage_dataset_storage_baseurl }}
 fcrepo.url={{ easy_stage_dataset_fcrepo_service_url }}
 fcrepo.user={{ easy_stage_dataset_fcrepo_user }}
 fcrepo.password={{ easy_stage_dataset_fcrepo_password }}
 db-connection-url={{ easy_stage_dataset_db_connection_url }}
+redirect-unset-url=http://unset.dans.knaw.nl
```

Comment:
Utter laziness on my part. Yes a placeholder would be better. Howver, this is probably not something that is going to vary a lot. Please, feel free to add the placeholder :+1: